.pylintrc

```diff
@@ -17,10 +17,10 @@ docstring-quote=double
 [BASIC]
 
 # Regular expression which should only match correct function names
-function-rgx=^[_a-z][a-z0-9_]*$
+function-rgx=^[a-z_][a-z0-9]+(_[a-z0-9]+)*$
```

Comment:
You don't need to change this one, You only need to change regex for variables. Revert it back to its original form.

Reply:
Done. Reverted back to original.